stonesoup/sensor/base.py

```diff
@@ -19,3 +20,36 @@ class Sensor(Base):
     def gen_measurement(**kwargs):
         """Generate a measurement"""
         raise NotImplementedError
+
+
+class Sensor3DCartesian(Sensor):
+    """Sensor base class extended to include 3D cartesian motion
+
+
+    """
+    position = Property(StateVector,
+                        doc="The sensor position on a 3D Cartesian plane,\
+                            expressed as a 3x1 array of Cartesian \
+                            coordinates in the order :math:`x,y,z`")
+    orientation = Property(
+        StateVector,
+        doc="A 3x1 array of angles (rad), specifying the sensor \
+            orientation in terms of the counter-clockwise rotation \
+            around each Cartesian axis in the order :math:`x,y,z`. The \
+            rotation angles are positive if the rotation is in the \
+            counter-clockwise direction when viewed by an observer \
+            looking along the respective rotation axis, towards the \
+            origin")
+
```

Comment:
Does it matter what coordinate system is used? (East, North, Up or North, East, Down,  etc)